data_pipeline.py

```diff
@@ -0,0 +1,264 @@
+import tensorflow as tf
+import numpy as np
+from projection_utils import get_axismatrix, projection_to_rgb
+import os
+
+ROOT_DIR = '/home/chidubem/deep-3d-reconstruction/scannet/scans/'
+BATCH_SIZE = 5
+NUM_EPOCHS = 5
+
+
+def _bytes_feature(value):
+    """Returns a bytes_list from a string / byte."""
+    if isinstance(value, type(tf.constant(0))):
+        # BytesList won't unpack a string from an EagerTensor.
+        value = value.numpy()
+    return tf.train.Feature(bytes_list=tf.train.BytesList(value=[value]))
+
+
+def _float_feature(value):
+    """Returns a float_list from a float / double."""
+    return tf.train.Feature(float_list=tf.train.FloatList(value=[value]))
+
+
+def _int64_feature(value):
+    """Returns an int64_list from a bool / enum / int / uint."""
+    return tf.train.Feature(int64_list=tf.train.Int64List(value=[value]))
+
+
+def serialize_example(
+        images,
+        depths,
+        extrinsic_matrix,
+        intrinsic_matrix,
+        axis_matrix,
+        box_objects):
+    """Serialize to tensoflow example and save to tf records
+    Args:
+        images: array of rgb images for each scene
+        depths: array of depth images for each scene
+        extrinsic_matrix: (4x4 array): camera to world extrinsic parameters
+        intrinsic_matrix: (4x4 array): camera intrinsic parameters
+        axis_alinment: (4x4 array): axis_alignment matrix
+        box_objects: Nx7 array containing box center, size and angle of N boxes
+    Return: serialized tensorflow example of a scene
+    """
+
+    feature = {
+        'images': _bytes_feature(images),
```

Comment:
Add a comment, if you expect this to contains multiple images. Also specify what is the type of the image (uint8 opencv or numpy array)